### ISSUE TITLE
Not working on Node 6.0.0

### ISSUE THREAD

**Issue by Gared** [ether/etherpad-lite#2956]
When I try to start etherpad with node version 6.0.0 I get the following exception:
```
/home/gared/dev/etherpad-lite/node_modules/ep_etherpad-lite/node_modules/npm/lib/npm.js:31
    throw new Error('npm.load() required')
    ^

Error: npm.load() required
    at Object.npm.config.get (/home/gared/dev/etherpad-lite/node_modules/ep_etherpad-lite/node_modules/npm/lib/npm.js:31:11)
    at EventEmitter.Object.defineProperty.get [as dir] (/home/gared/dev/etherpad-lite/node_modules/ep_etherpad-lite/node_modules/npm/lib/npm.js:410:22)
    at Object.exports.getPackages (/home/gared/dev/etherpad-lite/node_modules/ep_etherpad-lite/static/js/pluginfw/plugins.js:106:29)
    at Object.exports.update (/home/gared/dev/etherpad-lite/node_modules/ep_etherpad-lite/static/js/pluginfw/plugins.js:83:11)
    at /home/gared/dev/etherpad-lite/src/node/server.js:66:13
    at fn (/home/gared/dev/etherpad-lite/src/node_modules/async/lib/async.js:641:34)
    at Immediate._onImmediate (/home/gared/dev/etherpad-lite/src/node_modules/async/lib/async.js:557:34)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)

```

**Comment by rugk** [comment]
I get this too, but I also get this with Node v6:
```
[time...] [ERROR] console - (node:8092) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[time...] [WARN] console - DirtyDB is used. This is fine for testing but not recommended for production.
[time...] [ERROR] console - (node:8092) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
[time...] [ERROR] console - (node:8092) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
/home/etherpad/node_modules/ep_etherpad-lite/node_modules/npm/lib/npm.js:31
    throw new Error('npm.load() required')
```

Is this also caused by Node 6?

`graceful-fs` is quite up-to-date AFAIK:
```shell
$ npm install graceful-fs
/home/etherpad
└── graceful-fs@4.1.3 
```

**Comment by Gared** [comment]
Probably the reason is this change: https://github.com/nodejs/node/pull/5950
@rugk This error appears in older versions of node.js too. The package is outdated and you were probably in the wrong directory doing the npm install (there is a src directory with a node_modules in it)

**Comment by rugk** [comment]
> @rugk This error appears in older versions of node.js too. The package is outdated and you were probably in the wrong directory doing the npm install (there is a src directory with a node_modules in it)

Okay, anyway I could solve it with a downgrade to Node v5.

**Comment by twinshadow** [comment]
I can confirm that a downgrade to node 5.9.0 did work around this issue. To do this with homebrew on OSX 10.10.5, running `brew switch node 5.9.0` and `brew pin 5.9.0` should keep the workaround in place.

**Comment by Gared** [comment]
Etherpad is working again with Node v7 (tested with latest nightly)

**Comment by dvzrv** [comment]
This seems to be resolved with nodejs>=6.2

**Comment by Treverix** [comment]
I can confirm that. I had the same error, received node 6.2.1 after a brew upgrade and now it works.
